plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1262,6 +1261,27 @@ public void testDeleteWithVarcharPredicate()
         throw new SkipException("This is implemented by testDeleteWithVarcharEqualityPredicate");
     }
 
+    @Test
+    public void testInsertWithoutTemporaryTable()
+    {
+        if (!hasBehavior(SUPPORTS_CREATE_TABLE)) {
+            throw new SkipException("CREATE TABLE is required for testing non-transactional write support");
+        }
+        Session session = Session.builder(getSession())
+                .setCatalogSessionProperty(getSession().getCatalog().orElseThrow(), "use_temporary_table_on_insert", "false")
+                .build();
+
+        int numberOfRows = 1000;
```

Comment:
why 1000?

Reply:
It's irrelevant but I've raised it to 5000 to it's larger than the default batch size insert.